plugins/kotlin/idea/tests/testData/quickfix/override/nothingToOverride/addFunctionWithoutDefaultValue.kt.after

```diff
@@ -0,0 +1,9 @@
+// "Add 'abstract fun foo(x: String)' to 'I'" "true"
+// ERROR: An overriding function is not allowed to specify default values for its parameters
+interface I {
+    fun foo(x: String)
+}
+
+class Foo : I {
+    override fun foo(x: String = "") {}
```

Comment:
I'd expect the refactoring to generate default values in the base class/interface because override declarations are prohibited to have default parameters. Like this:
```
interface I {
    fun foo(x: String)
}

class Foo : I{
    override fun foo(x: String = "") {}
}
```